card_lookup.py

```diff
@@ -24,11 +24,19 @@ def clean_disc_string(string):
 def process_card_lookup(matches):
     with DBConnect(settings.CARD_DB) as conn:
         cur = conn.cursor()
-        sql = 'SELECT * FROM cards WHERE card_name = ? COLLATE NOCASE'
+        # sql = 'SELECT * FROM cards WHERE card_name = ? COLLATE NOCASE'
+        sql = "SELECT * FROM cards WHERE card_name LIKE '%?%'"
```

Comment:
```suggestion
        sql = "SELECT * FROM cards WHERE card_name LIKE ?"
```

The '' surrounding the ? needs to be removed. With them, it interprets this as a literal value, not a placeholder.

Second, the % % can't be placed here. It needs to be part of the value to be interpolated (see suggestion further below). See: https://stackoverflow.com/questions/29849754/python-sqlite-how-do-you-do-partial-searches-using-placeholders/29849829 for reference.